storage/aptosdb/src/state_store/mod.rs

```diff
@@ -176,42 +176,42 @@ impl StateStore {
         Ok(result)
     }
 
+    /// Read the value for a specific version and require its existence. Call `get_value_by_value`
+    /// for most use cases.
     pub fn get_value_at_version(
```

Comment:
nit: hmm, this is confusing...  how about renaming it to "get_value_updated_at_version()"